addons/web_editor/static/src/js/widgets/widgets.js

```diff
@@ -1302,11 +1317,7 @@ var MediaDialog = Dialog.extend({
      * @private
      */
     _onSearchInput: function (ev) {
```

Comment:
Is this handler still usefull ? Is not the search entirely removed ?

Reply:
Yes it is. Why do you think it is useless ? _onSearchInput (MediaDialog) is used in each widget not only in ImageWidget

Reply:
Well it cannot be useful... there is no input in the media dialog anymore, only in its internal widgets. So this has nothing to do in the MediaDialog :/

Reply:
@qsm-odoo Can this be in the MediaWidget then ? Why was it in the MediaDialog before ?